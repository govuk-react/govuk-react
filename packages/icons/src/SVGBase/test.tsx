import * as React from 'react';
import ReactDOM from 'react-dom';
import { mount } from 'enzyme';
import SVG from '.';

const wrapper = <SVG>Example</SVG>;

describe('SVG', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(wrapper, div);
  });

  it('matches wrapper snapshot', () => {
    expect(mount(wrapper)).toMatchSnapshot('wrapper mount');
  });
});
