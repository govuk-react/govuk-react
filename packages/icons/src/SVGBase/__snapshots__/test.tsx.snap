// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVG matches wrapper snapshot: wrapper mount 1`] = `
<SVG
  fill="currentColor"
  width="100%"
>
  <svg
    fill="currentColor"
    height="100%"
    style={
      Object {
        "display": "block",
      }
    }
    version="1.1"
    width="100%"
  >
    <title />
    Example
  </svg>
</SVG>
`;
